app/src/main/java/com/marverenic/music/player/MusicPlayerExtension.java

```diff
@@ -0,0 +1,10 @@
+package com.marverenic.music.player;
+
+import com.marverenic.music.model.Song;
+
+public abstract class MusicPlayerExtension {
+    void onSongStart(Song song) {}
```

Comment:
Can we rename this to `onSongStarted` to be consistent with the rest of the tenses here?